drivers/hdc1000/hdc1000.c

```diff
@@ -33,6 +33,20 @@
 
 #define I2C_SPEED                  I2C_SPEED_FAST
 
+#ifndef HDC1000_RENEW_INTERVAL
+#define HDC1000_RENEW_INTERVAL 5000000ul
```

Comment:
Where did you take this value from? Is there a max. request interval for this device?

Reply:
No. It basically depends on user's preference. If I really want to get real-time temp/hum data, I will set this value to 0. If I'm OK with getting the temp/hum data 5 seconds ago, I will set this value to 5 seconds.

Reply:
I would vote to go per default with the maximum sampling rate that the sensor can provided, probably something around 1-2Hz for this kind of sensor? (haven't checked the datasheet...)

Reply:
I don't see maximum sampling rate from the datasheet. But 1 second seems to be reasonable.